src/util.cc

```diff
@@ -647,6 +647,19 @@ RAIIIsolate::~RAIIIsolate() {
   isolate_->Dispose();
 }
 
+#ifdef _WIN32
```

Comment:
You don't need it. We have a `node::kPathSeparator` so you can just compare `if (c == node::kPathSeparator)`

Reply:
From what I saw in JS implementation for windows we accept both forward and back slash separator. Does `node::kPathSeparator` accept both or is one of them not needed?

Reply:
node::kPathSeparator is chooosed based on the platform, so yes.